lib/controller.js

```diff
@@ -92,7 +95,7 @@ Tessel.list = function(opts) {
 
 Tessel.get = function(opts) {
   return new Promise(function(resolve, reject) {
-    logs.info('Looking for your Tessel...');
+    logs.info('Searching nearby LAN (netmask) and USB for Tessels...');
```

Comment:
> I think it is more useful to get an info about where and what is searched... 

Any tessel-centric command allows for `--usb` or `--lan` flags, so this message won't make sense if one of those is used.